### DISCUSSION TITLE
multi-tap and hold

### DISCUSSION THREAD

**Discussion by ghistes** [kmonad/kmonad#929] [Q&A]
Hi,

I cannot figure out how to do the following:

I want ralt to send f11 when tapped once, f12 when tapped twice but still act as an ralt key when held.

I have tried all sorts of variations of combining multi-tap with other tap-functions but either I am doing something wrong or what I want is not really possible...

Could someone give me a tip here?

**Comment by jokesper** [comment]
Something linke `multi-tap 200 (tap-hold 1 f11 ralt) f12`
Should work, though it's probably a bit whanky, since
1. the timeouts get added and
2. `multi-tap` ignores other keys and focuses only on the timeout (just like `tap-hold`).

I don't quite remember if 2 is true. If it is, it could make the holding feel unnatural. Otherwise it would behave like a `tap-next` (or similar button). This would interact with the timeouts and button order wierdly.

`multi-tap` holding behavior is broken in v0.4.3. #897 fixed this.
You should you either a git version newer than this or 0.4.2 (Has quite some fixes missing, but I believe it wasn't broken then)
